interfaces/mount/entry.go

```diff
@@ -168,3 +169,24 @@ func LoadFSTab(reader io.Reader) ([]Entry, error) {
 	}
 	return entries, nil
 }
+
+// SaveFSTab writes a list of entries to a fstab-like file.
+//
+// The supported format is described by fstab(5).
+//
+// Note that there is no support for comments, both the LoadFSTab function and
+// SaveFSTab just ignore them.
+//
+// Note that there is no attempt to use atomic file write/rename tricks. The
```

Comment:
First, I don't think this is true. These files also go into /var/lib/snapd/mount, rigth?

Second, even if this is true, a snapd abrupt termination would leave the system in a bad state.

Then, how come this was merged without getting the discussion getting to a conclusion?